src/components/Restoration/ExtendTimeLimit.vue

```diff
@@ -0,0 +1,131 @@
+    >
+      <template v-if="approvalType == ApprovalTypes.VIA_COURT_ORDER">
+        <ApprovalType
+          :courtOrderNumber="courtOrderNumberText"
+          :isCourtOrderOnly="true"
+          :isCourtOrderRadio="false"
+          :invalidSection="invalidSectionApprovalType"
+          @courtNumberChange="setRestorationCourtOrder({ fileNumber: $event })"
+          @valid="setApprovalTypeValid($event)"
+        />
+      </template>
+    </section>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Action, Getter } from 'vuex-class'
+import { ApprovalType } from '@bcrs-shared-components/approval-type'
+import { ApprovalTypes, RestorationTypes } from '@/enums'
+import Actions from '@/components/common/Actions.vue'
+import { Component, Watch } from 'vue-property-decorator'
+import { DateMixin, CommonMixin } from '@/mixins'
+import DateUtilities from '@/services/date-utilities'
+import { LegalServices } from '@/services/'
+import { LimitedRestorationPanel } from '@bcrs-shared-components/limited-restoration-panel'
+import { RestorationStateIF, StateFilingRestorationIF } from '@/interfaces'
+
+@Component({
+  mixins: [
+    CommonMixin,
+    DateMixin
+  ],
+  components: {
+    Actions,
+    ApprovalType,
+    LimitedRestorationPanel
+  }
+})
+export default class ExtendTimeLimit extends Vue {
+  @Getter getApprovalTypeValid!: boolean
+  @Getter getCurrentDate!: string
+  @Getter getExpiryValid!: boolean
+  @Getter getRestoration!: RestorationStateIF
+  @Getter getStateFilingRestoration!: StateFilingRestorationIF
+
+  @Action setApprovalTypeValid!: ActionBindingIF
+  @Action setExpiryValid!: ActionBindingIF
+  @Action setRestorationCourtOrder!: ActionBindingIF
+  @Action setRestorationExpiry!: ActionBindingIF
+  @Action setValidComponent!: ActionBindingIF
+
+  // Enum for template
+  readonly ApprovalTypes = ApprovalTypes
+  readonly RestorationTypes = RestorationTypes
+
+  /** The limited restoration state filing's approval type. */
+  get approvalType (): ApprovalTypes {
+    return this.getStateFilingRestoration.approvalType
+  }
+
+  /** The court order draft file number. */
+  get courtOrderNumberText (): string {
+    return this.getRestoration.courtOrder?.fileNumber || ''
+  }
+
+  /** The expiry draft date for extension. */
+  get expiry (): string {
+    return DateUtilities.subtractMonthsToDate(this.previousNumberOfMonths, this.getRestoration.expiry)
+  }
+
+  /** This extension time's validity state (when prompted by app). */
+  get invalidSectionExpiry (): boolean {
+    return !this.getExpiryValid
+  }
+
+  /** This approval type's validity state (when prompted by app). */
+  get invalidSectionApprovalType (): boolean {
+    return !this.getApprovalTypeValid
+  }
+
+  /** The previously selected number of months of limited restoration. */
+  get previousNumberOfMonths (): string {
+    return this.subtractDates(this.getCurrentDate, this.getStateFilingRestoration.expiry)
```

Comment:
This will give you a different result as currentDate changes to the next month, so it's not what the comment says it is.

It _might_ be the remaining months to the original expiry date.